stake-pool/program/src/instruction.rs

```diff
@@ -36,56 +36,104 @@ pub enum StakePoolInstruction {
     ///
     ///   0. `[w]` New StakePool to create.
     ///   1. `[]` Owner
-    ///   2. `[]` pool token Mint. Must be non zero, owned by withdraw authority.
-    ///   3. `[]` Pool Account to deposit the generated fee for owner.
-    ///   4. `[]` Token program id
+    ///   2. `[w]` Uninitialized validator stake list storage account
+    ///   3. `[]` pool token Mint. Must be non zero, owned by withdraw authority.
+    ///   4. `[]` Pool Account to deposit the generated fee for owner.
```

Comment:
this can be done in a separate PR, but you can also pass the rent sysvar to make sure the validator stake list and stake pool are rent exempt

Reply:
Next PR it is :)